client/my-sites/checkout/checkout/index.jsx

```diff
@@ -437,6 +437,7 @@ export class Checkout extends React.Component {
 		// This tests the flow that was not eligible for G Suite
 		// There's an additional test above that tests directly aginst the G Suite upsell
 		if (
+			'offer' === abtest( 'conciergeUpsellDial' ) &&
```

Comment:
I know we're not using this as a real test, but just to make this a bit cleaner, can we just wrap this call to `abtest()` around `return /checkout/${ selectedSiteSlug }/add-quickstart-session/${ receiptId };`? Instead of adding it to the beginning of this block of conditions.

Otherwise it gets called first, before all the other eligibility checks. So the percentage we set won't be fully accurate and we'll be needlessly making the call for users who wouldn't have otherwise been eligible for the offer.

Reply:
Good catch on eligibility checks! 👍